streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -109,9 +109,9 @@ void createTasks(final Collection<TopicPartition> assignment) {
         active.closeNonAssignedSuspendedTasks(assignedActiveTasks);
         addStreamTasks(assignment);
         addStandbyTasks();
-        final Set<TopicPartition> partitions = active.uninitializedPartitions();
-        log.trace("Pausing partitions: {}", partitions);
-        consumer.pause(partitions);
+        // Pause all the partitions until the underlying state store is ready for all the active tasks.
+        log.trace("Pausing partitions: {}", assignment);
```

Comment:
I'm wondering if this should be `debug` or not, same below when resuming assignment, although I don't feel too strongly about this.

Reply:
Earlier, It was logged as `trace` so left it as it is.